app/src/main/java/net/bible/android/control/page/CurrentBiblePage.kt

```diff
@@ -202,12 +202,11 @@ class CurrentBiblePage(
         }
     }
 
-    //TODO allow japanese search - japanese bibles use smartcn which is not available
     /** can we enable the main menu search button
      */
     override val isSearchable: Boolean
-        get() = try { //TODO allow japanese search - japanese bibles use smartcn which is not available
-            !currentDocument!!.doesNotExist && "ja" != currentDocument!!.language.code
```

Comment:
- [ ] todo: check japanese documents

Reply:
I performed a quick check by copy-pasting part of the document and searching for it, which worked. But I don't speak/read Japanese, so I don't know if the results made any sense.

Reply:
I will try to consult my friend chatgpt. He speaks japanese for sure 😊